### ISSUE TITLE
VSCode User Area does not work

### ISSUE THREAD

**Issue by AlexTheOutsider22** [Dyzean/Tokyo-Night#20]
I took the @import from the theme, and it does not work at all, maybe due to the various CSS changes in Discord itself

**Comment by Dyzean** [comment]
Yeah this is known, i dont have time to maintain this theme anymore. so if anyone happen to know how to fix it just make a PR
